cubi_tk/isa_tpl/__init__.py

```diff
@@ -255,4 +258,21 @@ def run(args, parser, subparser):  # pragma: nocover
     if not args.tpl:  # pragma: nocover
         return run_nocmd(args, parser, subparser)
     else:
-        return args.isa_tpl_cmd(args, parser, subparser)
+        args.isa_tpl_cmd(args, parser, subparser)
+
+        # output validation
+        logger.info("Running AltamISA validator:")
+        i_files = Path(args.output_dir).rglob("i_*")
+        args.show_duplicate_warnings = False
+        warnings.filterwarnings("error", category=altamisa.exceptions.CriticalIsaValidationWarning)
+        for i in i_files:
+            try:
+                with i.open() as i_file:
+                    args.input_investigation_file = i_file
+                    altamisa.apps.isatab_validate.run(args)
+            except (
+                altamisa.exceptions.ParseIsatabException,
+                altamisa.exceptions.CriticalIsaValidationWarning,
+            ):
+                shutil.rmtree(args.output_dir)
+                raise
```

Comment:
Additionally this function now does not seem to have return value anymore? I'm not sure if that can lead to errors with the underlying code or if its 'only' a consistency issue, but either way this function should probably still have a return value (iirc cubi-tk simply retruns False/True or 0/1 as error codes).